files/install_graphdb.sh

```diff
@@ -62,6 +62,7 @@ chown -R graphdb:graphdb /etc/graphdb \
 # Configure systemd for GraphDB and GraphDB proxy
 mv /tmp/graphdb-cluster-proxy.service /lib/systemd/system/graphdb-cluster-proxy.service
 mv /tmp/graphdb.service /lib/systemd/system/graphdb.service
+mv /tmp/gdb_environments /etc/environment.d/gdb_environments
```

Comment:
see [this](https://github.com/Ontotext-AD/terraform-aws-graphdb/pull/21/files#r1389356662) comment and change if needed.